webapps/aqstats/config.json

```diff
@@ -0,0 +1 @@
+/home/zotya/work/eea.elasticsearch/etc/dev/aqstats/config.json
```

Comment:
This shouldn't be checked into git as a config.
Can you use the pattern in eea_search.js to have the config in the same directory as the application